packages/pds/src/api/com/atproto/server/util.ts

```diff
@@ -28,21 +28,29 @@ export const getRandomToken = () => {
   return token.slice(0, 5) + '-' + token.slice(5, 10)
 }
 
-// @TODO once supporting multiple pdses, validate pds in did doc based on allow-list.
-export const didDocForSession = async (
+export const safeResolveDidDoc = async (
   ctx: AppContext,
   did: string,
   forceRefresh?: boolean,
 ): Promise<DidDocument | undefined> => {
-  if (!ctx.cfg.identity.enableDidDocWithSession) return
   try {
     const didDoc = await ctx.idResolver.did.resolve(did, forceRefresh)
     return didDoc ?? undefined
   } catch (err) {
-    dbLogger.warn({ err, did }, 'failed to resolve did doc')
+    httpLogger.warn({ err, did }, 'failed to resolve did doc')
   }
 }
 
+// @TODO once supporting multiple pdses, validate pds in did doc based on allow-list.
```

Comment:
Not sure if I understand this TODO anymore 🤔 could it have been meant for entryway?

Reply:
yeah p sure it's meant for entryway 👍